.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -29,14 +29,15 @@ jobs:
       - name: Publish the new helm chart and update index.yaml
         run: |
           helm package cromwell-helm/
-          mv cromwell-${NEXT_VERSION}.tgz charts/cromwell-${NEXT_VERSION}.tgz
+          mv cromwell-${NEXT_VERSION}.tgz charts
           helm repo index --url https://broadinstitute.github.io/cromwhelm/charts/ charts
 
       - name: Push changes to GitHub
         env:
           GITHUB_TOKEN: ${{ secrets._GITHUB_TOKEN }}
         run: |
           git checkout main
+          git add charts/cromwell-${NEXT_VERSION}.tgz
```

Comment:
On my workstation I typically use `git add .` to avoid missing things, could be applicable here (or there could be files we specifically don't want, in which case ignore)

Reply:
Yeah I also usually do `git add .`, but I thought about being explicit here. But it does make sense to use `git add .` here as well just to be safe. I will update and test it out.